docker/Dockerfile-pgtests

```diff
@@ -1,12 +1,21 @@
 # Use the Sytest image that comes with a lot of the build dependencies
 # pre-installed
-FROM matrixdotorg/sytest:latest
+FROM matrixdotorg/sytest:bionic
 
 # The Sytest image doesn't come with python, so install that
 RUN apt-get update && apt-get -qq install -y python3 python3-dev python3-pip
 
 # We need tox to run the tests in run_pg_tests.sh
 RUN python3 -m pip install tox
 
+# Initialise the db
+RUN su -c '/usr/lib/postgresql/10/bin/initdb -D /var/lib/postgresql/data -E "UTF-8" --lc-collate="en_US.UTF-8" --lc-ctype="en_US.UTF-8" --username=postgres' postgres
```

Comment:
why American locales? Don't we definitely want `C` as the locale? (Synapse will refuse to set up databases in the wrong locale.)

Perhaps this doesn't matter here (maybe this is just the `postgres` management database?) but it strikes me as odd that we'd specify a specific locale if we don't care about it.

Reply:
Oh hmm you nicked it from `run_pg_tests.sh`. Odd, but maybe we shouldn't remove it after all if we don't know why it's there

Reply:
Yeah, I just transplanted this into the dockerfile. Even if we don't know why it's there, I think this should probably be `C`. (No-one's used this in ages anyway!)